build-pyspy.bat

```diff
@@ -0,0 +1,56 @@
+@echo off
+WHERE cargo
+IF %ERRORLEVEL% NEQ 0 (
+	ECHO cargo wasn't found. Attempting to install...
+	.\dep\rustup-init.exe -y
+)
+
+start /wait .\dep\vs_BuildTools.exe --passive --add Microsoft.VisualStudio.Workload.VCTools --add Microsoft.VisualStudio.Component.Windows10SDK.19041 --add Microsoft.VisualStudio.Component.VC.Tools.x86.x64
+
+SET LINK_LOC="C:\Program Files (x86)\Microsoft Visual Studio\2022\BuildTools\VC\Tools\MSVC\14.33.31629\bin\Hostx86\x86\link.exe"
+
+SET SDK_LOC="C:\Program Files (x86)\Windows Kits\10\Lib\10.0.19041.0\um\x86\advapi32.lib"
```

Comment:
Can't these 2 paths change? (e.g `14.33.31629` looks like a minor version that can be changed without any change on our side?)

Reply:
I made some changes to search for advapi32.lib and link.exe instead of hardcoding the path

Reply:
Done